### PR TITLE
Wagtail 2.0  compatibility!

### PR DESCRIPTION
**Pull request by harrislapiroff**

### CODE REVIEW

**Comment by harrislapiroff** [comment]
This will be more satisfying once we have tests + tox.

**Comment by harrislapiroff** [comment]
@emilyhorsman if you have time, this is for your eyes

File: wagtailautocomplete/edit_handlers.py [hunk 1]
```diff
@@ -16,23 +21,42 @@ def _can_create(page_type):
...
-        self.page_type = page_type
-        self.is_single = is_single
-
-    def bind_to_model(self, model):
-        can_create = _can_create(self.page_type)
-        base = dict(
-            model=model,
-            field_name=self.field_name,
-            widget=type(
+if 'BaseFieldPanel' in locals():
```
Comment (emilyhorsman) [inline wagtailautocomplete/edit_handlers.py]: Is there a more robust or canonical way to perform this check? Does the wagtail package export a version constant?

File: wagtailautocomplete/urls/admin.py [hunk 2]
```diff
@@ -1,5 +1,11 @@
 from django.conf.urls import url
-from wagtail.wagtailadmin.decorators import require_admin_access
+
+try:
+    # Wagtail 2.x
+    from wagtail.admin.decorators import require_admin_access
```
Comment (emilyhorsman) [inline wagtailautocomplete/urls/admin.py]: omg dropping the `wagtail` prefix 😍

File: wagtailautocomplete/edit_handlers.py [hunk 3]
```diff
@@ -16,23 +21,42 @@ def _can_create(page_type):
...
+    # Wagtail 2.x
+    class AutocompletePanel(FieldPanel):
+        def __init__(self, field_name, page_type='wagtailcore.Page', is_single=True, **kwargs):
+            super().__init__(field_name, **kwargs)
+            # is_single defaults to True in order to have easy drop-in
+            # compatibility with wagtailadmin.edit_handlers.PageChooserPanel.
+            self.page_type = page_type
+            self.is_single = is_single
+
+        def on_model_bound(self):
```
Comment (emilyhorsman) [inline wagtailautocomplete/edit_handlers.py]: Ohh, that's slightly better than `bind_to_model` :D

File: wagtailautocomplete/edit_handlers.py (hunk 1 above)
Comment (emilyhorsman) [reply wagtailautocomplete/edit_handlers.py]: oh jk you figured this out in the next commit

File: wagtailautocomplete/edit_handlers.py [hunk 4]
```diff
@@ -21,8 +16,10 @@ def _can_create(page_type):
     ))
 
 
-if 'BaseFieldPanel' in locals():
+if VERSION < (2, 0):
```
Comment (emilyhorsman) [inline wagtailautocomplete/edit_handlers.py]: +1

File: docs/getting_started.rst [hunk 5]
```diff
@@ -7,10 +7,6 @@ Requirements
 
 Wagtail Autocomplete requires Wagtail version 1.11 or newer.
 
-.. warning::
```
Comment (emilyhorsman) [inline docs/getting_started.rst]: 🎉

**Comment by emilyhorsman** [comment]
I'd wait for tests until a package release (do we even have those lol) but this is good to go 👍

**Pull request merged**